### ISSUE TITLE
editable prop doesn't work correctly 

### ISSUE THREAD

**Issue by Chathula** [halilb/react-native-textinput-effects#30]
i disabled editable,but yet when i click the left side of TextInput it popup(show) the keyboard.

why is that. need a Quick Fix!

```jsx
  render() {
    const { container, innerContainer, instructionText, inputMarginTop } = styles;

    return (
      <View style={container}>
        <View>
          <Text style={instructionText}> MyCarParks will send a one time SMS message to verify your phone number. Carrier SMS charges may apply. </Text>
        </View>
        <ScrollView style={innerContainer}>
          <Kohana
            style={{ backgroundColor: 'rgba(93, 116, 174, 0.2)' }}
            label={'0121 234 5678'}
            iconClass={FontAwesomeIcon}
            iconName={'phone'}
            iconColor={'#ddd'}
            iconColor={'#5D74AE'}
            labelStyle={{ color: '#5D74AE', fontWeight: '300' }}
            inputStyle={{ color: '#5D74AE' }}
            onChangeText={this.onMobileChange.bind(this)}
            editable={false}
            value={this.state.text}
          />
        <Text style={{ textAlign: 'center', fontWeight: '100', marginTop: 5 }}>We'll send a text to verify your phone</Text>

        <View style={{ marginTop: 80, marginBottom: 10 }}>
            <Keyboard
                keyboardType="decimal-pad"
                onClear={this.handleClear.bind(this)}
                onDelete={this.handleDelete.bind(this)}
                onKeyPress={this.handleKeyPress.bind(this)}
                disableDot={true}
                disableOtherText={true}
                disableBorder={true}
                disableClearButtonBackground={false}
            />
        </View>

        <TouchableOpacity style={[inputMarginTop, styles.button]} onPress={this.onButtonPress.bind(this)}>
            <Text style={styles.buttonText}> NEXT </Text>
        </TouchableOpacity>

        </ScrollView>
      </View>
    );
  }
```

**Comment by Chathula** [comment]
is there anyone here had this issue?

**Comment by Chathula** [comment]
@halilb have u seen this issue?

**Comment by ElieMyIdea** [comment]
Same issue here on IOS with RN 0.38.0. Need fix soon

**Comment by RiusmaX** [comment]
+1 same problem on iOS but not on Android with RN 0.38.

**Comment by halilb** [comment]
Thanks for reporting guys!

I've just published 0.3.1 which contains a fix for this problem. `editable=false` should be enough to disable all touchable areas now. Let me know if it doesn't workout for you @Chathula @ElieMyIdea @RiusmaX
